ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -82,18 +82,21 @@ export const ApplicationStatusPanel = ({application, showOperation, showConditio
                 <React.Fragment>
                     {sectionHeader(
                         {
-                            title: 'CURRENT SYNC STATUS',
+                            title: 'SYNC STATUS',
                             helpContent: 'Whether or not the version of your app is up to date with your repo. You may wish to sync your app if it is out-of-sync.'
                         },
                         hasMultipleSources,
                         source.chart ? null : () => showMetadataInfo(application.status.sync ? application.status.sync.revision : '')
                     )}
-                    <div className='application-status-panel__item-value'>
+                    <div className={`application-status-panel__item-value application-status-panel__item-value--${appOperationState.phase}`}>
                         <div>
                             <ComparisonStatusIcon status={application.status.sync.status} label={true} />
                         </div>
                         <div className='application-status-panel__item-value__revision show-for-large'>{syncStatusMessage(application)}</div>
                     </div>
+                    <div className='application-status-panel__item-name' style={{marginBottom: '0.5em'}}>
```

Comment:
I suppose the "When:" label here does not apply. The message displayed is one of the two on line 98.  So, perhaps the design should be revisited because the message appears to not be aligned / balanced with the Author and Comment information below it.  (IMHO).

Reply:
I'll reach out to the designer and we can revisit it - I think for the scope of this PR it's okay to leave as is